manifests/init.pp

```diff
@@ -150,7 +150,7 @@
   $service_ensure                                            = running,
   $service_flags                                             = undef,
   $service_restart                                           = undef,
-  $service_name                                              = undef,
+  $service_name                                              = "nginx",
```

Comment:
This should be in single quotes

Reply:
Why is that - for security, to stop string evaluation?

Reply:
It's a lint rule we have. Strings without variables should be in single quotes. I think it was copied from ruby.

Reply:
If you can fix this then the build will pass and I think it's good to go.

Reply:
Done with https://github.com/voxpupuli/puppet-nginx/pull/1159/commits/e557db0dc410d23525731aea15bd6530899395b4